### DISCUSSION TITLE
Feature Request: VSCodium support

### DISCUSSION THREAD

**Discussion by asgeirn** [coder/coder#15846] [Feature Request]
We're investigating using Coder in an isolated environment with no Internet connectivity, and according to the [documentation for offline deployment](https://coder.com/docs/install/offline#microsoft-vs-code-remote---ssh), VSCode requires outbound port 443 access for the remote SSH extension to work.

I tried installing the Coder VSIX in VSCodium, but it fails because it depends on the ms-vscode-remote.remote-ssh extension.  According to https://github.com/VSCodium/vscodium/discussions/693 the extension 'jeanp413.open-remote-ssh' does this.

Would it be possible to provide a variant of the Codium extension on Open VSX for this, and a module for creating "Open in VSCodium" links in workspaces?

**Comment by matifali** [comment]
@code-asher We have now requests for VSCodium, [Windsurf](https://github.com/coder/modules/pull/353) and a buggy behavior on Cursor too.
We may need to ship a new separate binary without relying on the MS remote SSh extension as it only works on the MS version of VSCode.

cc: @bpmct

**Reply by code-asher** [reply]
Yeah it would be good idea to publish a version to Open VSX that uses `open-remote-ssh` instead.

**Reply by matifali** [reply]
We are tracking that in coder/vscode-coder#408

**Reply by matifali** [reply]
Coder extension now works with VSCodieum. We also publish to OpenVSX: https://open-vsx.org/extension/ms-vscode-remote/remote-ssh
